### PR TITLE
Feat(dashpay): upgrade to support Platform 0.22

### PR DESCRIPTION
**Pull request by HashEngineering**
## How Has This Been Tested?

- [ ] QA (Mobile Team)

## Checklist:

- [ ] I have performed a self-review of my own code and added comments where necessary
- [ ] I have added or updated relevant unit/integration/functional/e2e tests

### CODE REVIEW

**Comment by HashEngineering** [comment]
Use a merge commit to merge.

File: wallet/src/de/schildbach/wallet/ui/AboutActivity.kt [hunk 1]
```diff
@@ -84,7 +84,7 @@ class AboutActivity : BaseMenuActivity(), SensorEventListener {
 
     private fun showFirebaseInstallationId() {
         FirebaseInstallations.getInstance().id.addOnCompleteListener { task ->
-            firebase_installation_id.isVisible = task.isSuccessful && BuildConfig.DEBUG
+            firebase_installation_id.isVisible = task.isSuccessful
```
Comment (Syn-McJ) [inline wallet/src/de/schildbach/wallet/ui/AboutActivity.kt:87]: Do we want to show the firebase installation id in production?

**Review by Syn-McJ — APPROVED** [review]
Looks good

File: wallet/src/de/schildbach/wallet/ui/AboutActivity.kt (hunk 1 above)
Comment (HashEngineering) [reply wallet/src/de/schildbach/wallet/ui/AboutActivity.kt:87]: Yes, it was requested to make it easier to test In-App-Messaging

**Pull request merged**